### PR TITLE
Close Navbar on mobile

### PR DESCRIPTION
**Pull request by vitalii-ilchenko-ssa-group**

### CODE REVIEW

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 0       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 2  | ❗       |
    


*❗ storybook/utility-responsive-image--responsive-image__[w1440px]*: 


| Before | After |
| --- | --- |
|  |  |



*❗ storybook/utility-responsive-image--responsive-image__[w1920px]*: 


| Before | After |
| --- | --- |
|  |  |

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 0       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 4  | ❗       |
    


*❗ custom/widgets-collapsiblenavbar-opened__[w899px]_statistics_submenu*: 


| Before | After |
| --- | --- |
|  |  |



*❗ custom/widgets-meal-nutrients--hint-opened__[w1440px]*: 


| Before | After |
| --- | --- |
|  |  |



*❗ custom/widgets-navbar-opened__[w1920px]*: 


| Before | After |
| --- | --- |
|  |  |



*❗ storybook/widgets-calories--default__[w1440px]*: 


| Before | After |
| --- | --- |
|  |  |

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 0       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 1  | ❗       |
    


*❗ storybook/utility-responsive-image--responsive-image__[w1920px]*: 


| Before | After |
| --- | --- |
|  |  |

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 0       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 3  | ❗       |
    


*❗ custom/widgets-collapsiblenavbar-opened__[w899px]_statistics_submenu*: 


| Before | After |
| --- | --- |
|  |  |



*❗ custom/widgets-meal-nutrients--hint-opened__[w390px]*: 


| Before | After |
| --- | --- |
|  |  |



*❗ custom/widgets-user-profile--is-opened__[w1920px]*: 


| Before | After |
| --- | --- |
|  |  |

File: packages/widgets/src/components/CollapsibleNavBar/CollapsibleNavBar.tsx [hunk 1]
```diff
@@ -55,9 +57,17 @@ export const CollapsibleNavBar = ({
...
+                key={keyName}
+                onClick={handleCloseMobileMenu}
+              />
             ) : (
-              <NavBarItemWithoutSubMenu item={item} key={keyName} />
+              <NavBarItemWithoutSubMenu
+                item={item}
+                key={keyName}
+                onClick={handleCloseMobileMenu}
+              />
```
Comment (react-developer-ssa-group) [inline packages/widgets/src/components/CollapsibleNavBar/CollapsibleNavBar.tsx:70]: It's better to optimize this condition, due to same properties.

**Review by react-developer-ssa-group — APPROVED** [review]

**Pull request merged**